pkg/apis/intevents/v1beta1/brokercell_defaults.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
```

Comment:
Should we start with v1alpha1? Or maybe this is an internal API so it's OK to change any time?

Reply:
I'm fine with either. I chose v1beta1 because Broker is also v1beta1, but as an internal API I think they have the same semantics. Making it alpha might be a useful shorthand to users  that the resource doesn't have a stable api. On the other hand, a beta resource (Broker) depending on an alpha resource might seem like a bad sign to a user.

Anyone have an opinion either way?

Reply:
At this point BrokerCell is an implementation detail and Broker API doesn't depend on it, right? (I assume we automatically assign a broker to a cell using annotation, we don't allow users to do so currently).

I'd like to make sure we can still easily change the BrokerCell API moving forward.

Reply:
I'll move it to v1alpha1. It might be useful to separates more mature resources like PullSubscription (which will presumably be v1beta1 in the same api group) from this one.